src/main/java/Car.java

```diff
@@ -1,9 +1,19 @@
 public class Car {
+    private static final String DEFAULT_NAME = "";
     private int position = 1;
     private final MovePolicy movePolicy;
+    private final String name;
 
     public Car(MovePolicy movePolicy) {
         this.movePolicy = movePolicy;
+        this.name = DEFAULT_NAME;
+    }
+
+    public Car(MovePolicy movePolicy, String name) {
+        validateName(name);
+
+        this.movePolicy = movePolicy;
+        this.name = name;
     }
 
     public void move() {
```

Comment:
```suggestion
    public void move(MovePolicy movePolicy) {
```
생성자를 통해 MovePolicy을 전달하고 있는데 메서드를 통해 전달하는 것은 어떨까?
두 가지 방법은 어느 방법으로 구현하는 것이 좋은 것이 있을까?

Reply:
1. 생성자를 통해 MovePolicy를 객체 생성 시에 지정
2. 메서드를 통해 전달 

자동차의 전진 방식은 변할수없다 와 같은 요구 사항이 있을 때는 1번을 통한 불변성 보장을,
불변성 보장이 필요없는 상황에서는 MovePolicy 메서드 전달을 통한 결합도 감소 측면에서 장점이 있다고 생각합니다 :) 

이번 미션에서는 후자가 유효하다고 생각해 제안주신 방법대로 리팩토링해보았습니다!